packages/plugins/graphql/server/services/utils/mappers/graphql-filters-to-strapi-query.js

```diff
@@ -68,13 +68,19 @@ module.exports = ({ strapi }) => {
 
       for (const [key, value] of Object.entries(filters)) {
         // If the key is an attribute, update the value
-        if (isAttribute(key)) {
+        if (isAttribute(key) || isComponent(attributes[key])) {
```

Comment:
Why do you need this check? A component attribute is by definition _an attribute_. Thus, you shouldn't need this extra condition.